pyfolio/plotting.py

```diff
@@ -29,6 +29,8 @@
 from . import timeseries
 from . import pos
 
+from .utils import BDAYS_PER_MONTH
```

Comment:
Can we call this APPROX_BDAYS_PER_MONTH?

Also, I understand from Justin that @twiecki thinks that BDAYS (vs TDAYS) is good, as that is the naming convention in pandas. However, doesn't pandas have full calendar functionality that we could use in this case? Why not pass rolling window as a pandas `datetime.timedelta` object with a given number of months or days and then let pandas or zipline tradingcalendar figure out the actual number of days? The use case would be something like
```python
plot_rolling_fama_french(returns, rolling_window=datetime.timedelta(months=3))
plot_rolling_fama_french(returns, rolling_window=datetime.timedelta(days=90))
```